ci/scripts/driver.sh

```diff
@@ -85,7 +82,7 @@ if [[ -s "${GFS_CI_ROOT}/${pr_list_file}" ]]; then
  pr_list=$(cat "${GFS_CI_ROOT}/${pr_list_file}")
 else
  echo "no PRs to process .. exit"
- exit
+ exit 1
```

Comment:
Why `exit 1` when there are no PR's to process?
Is that an error in the script?

Reply:
Oh sorry, right.